EIPS/eip-3475.md

```diff
@@ -1,18 +1,22 @@
 ---
 eip: 3475
-title: Multiple Callable Bonds
-author: Yohji Sakamoto (@sgmfinance)
-discussions-to: https://github.com/ethereum/EIPs/issues/3467
+title: ERC-3475 Multiple Callable Bonds Standard
+author: Yu LIU <info@debond.org>
```

Comment:
There must be at least one author with a GitHub handle.
```suggestion
author: Yu LIU (@sgmfinance)
```
```suggestion
author: Yu LIU (@sgmfinance), Yu LIU <info@debond.org>
```
Either of those are acceptable in this case.